src/jit/codegenxarch.cpp

```diff
@@ -1386,13 +1386,13 @@ void CodeGen::genCodeForDivMod(GenTreeOp* treeNode)
         }
         else
 #endif
-        if (dividend->gtRegNum != REG_RAX)
+            if (dividend->gtRegNum != REG_RAX)
```

Comment:
This is what jit-format does - I tried undoing it and it re-did it.

---
In reply to: [79524494](https://github.com/dotnet/coreclr/pull/7252#discussion_r79524494) [](ancestors = 79524494)